src/stores/auth.js

```diff
@@ -99,12 +94,6 @@ export const useAuthStore = defineStore('auth', {
           this.authSuccess(username);
         }
 
-        const gameStore = useGameStore();
-        if (isSpectating) {
-          const { gameId } = route.params;
-          gameStore.requestSpectate(Number(gameId));
-        }
```

Comment:
Finally removed this route-specific clutter so `requestStatus()` can be purely auth